lusidtools/cocoon/config/domain_settings.json

```diff
@@ -59,5 +59,16 @@
       "full_key_format": false,
       "required_call_attributes": [],
       "unique_attributes": []
+  },
+
+    "portfolio_group": {
+      "domain": "portfoliogroups",
+      "batch_allowed": true,
+      "default_batch_size": 2000,
```

Comment:
Is there a limit on the number of portfolios you can add in the creation request?